inject/src/test/groovy/io/micronaut/context/env/PropertySourcePropertyResolverSpec.groovy

```diff
@@ -99,14 +106,15 @@ class PropertySourcePropertyResolverSpec extends Specification {
         'my.property' | '${not.there:foo.bar:50}'                            | 'my.property' | String  | '10'
         'my.property' | '${not.there:also.not.there:50}'                     | 'my.property' | String  | '50'
         'my.property' | '${not.there:also.not.there:}'                       | 'my.property' | String  | ''
-        'my.property' | '${not.there:USER:50}'                               | 'my.property' | String  | System.getenv('USER')
```

Comment:
Is there a reason the lines with `USER` were deleted?

Reply:
tried to make it consistent with the other two environment variable cases.

Reply:
@graemerocher other than that using the system rules we have more control over environment variables, so I used it for all other cases.